nncf/onnx/quantization/quantizer_parameters.py

```diff
@@ -51,89 +47,39 @@ def get_level_low_level_high(tensor_type: np.dtype, num_bits: int) -> Tuple[int,
     return - (2 ** num_bits) // 2, (2 ** num_bits) // 2 - 1
 
 
-def calculate_scale_zero_point(max_val: np.ndarray, min_val: np.ndarray, level_low: int, level_high: int,
-                               mode: QuantizationMode, tensor_type: np.dtype) -> Tuple[np.ndarray, np.ndarray]:
+def calculate_scale_zero_point(input_low: np.ndarray, input_high: np.ndarray, level_low: int, level_high: int,
+                               mode: QuantizationMode, tensor_type: np.dtype,
+                               eps: float = 1e-6) -> Tuple[np.ndarray, np.ndarray]:
     """
     Calculates Quantizer/Dequantizer layer scale level.
-    Returns scale and zero_point values foe the quantizer.
+    Returns scale and zero_point values for the quantizer.
 
-    :param max_val: The maximum value of the input tensor.
-    :param min_val: The minimum value of the input tensor.
+    :param input_low: Minimum limit for input value.
+    :param input_high: Maximum limit for input value.
     :param level_low: The minimum level of quantizer quants.
     :param level_high: The maximum level of quantizer quants.
     :param mode: Symmetric or Asymmetric mode.
     :param tensor_type: Value type of the tensor. Could be INT8 or UINT8.
+    :param eps: The correction value is added to the input range to avoid division by zero.
     :return: Scale and Zero point values.
     """
+    input_range_safe = abs(level_high - level_low) + eps
     scale, zero_point = None, None
     if mode == QuantizationMode.SYMMETRIC:
-        input_abs_max = np.maximum(np.abs(max_val), np.abs(min_val))
-        max_val = input_abs_max
-        min_val = np.zeros_like(input_abs_max) if tensor_type == np.uint8 else -input_abs_max
-        scale = np.array((max_val - min_val) / float(level_high - level_low))
+        input_low = np.zeros_like(input_high) if tensor_type == np.uint8 else -input_high
+        scale = np.array((input_high - input_low) / (level_high - level_low))
```

Comment:
Don't we need to use `input_range_safe` to avoid division by zero?

Reply:
done